notes_project/notes_app/views.py

```diff
@@ -104,46 +90,11 @@ def user_logout(request):
 
 
 # Return something special for user
-
-
 @login_required
 def special(request):
     return HttpResponse("You are logged in, Nice!")
 
 
-# def user_login(request):
```

Comment:
You deleted this because we have login based views class?

Reply:
cleaning code